ipmiMgrApp/Db/sensor_ai_alias.template

```diff
@@ -1,5 +1,5 @@
 include "sensor_ai.db"
 
-alias("$(dev):$(prefix)$(attr)$(sensinst)", "$(P)$(R)$(prefix)$(attr)-Mon")
+alias("$(dev):$(prefix)$(attr)$(sensinst)", "$(AREA)$(DIS)$(dev):$(attr)-Mon")
 
-alias("$(dev):$(prefix)$(attr)$(sensinst)P", "$(P)$(R)$(prefix)$(attr)Prs-Sts")
+alias("$(dev):$(prefix)$(attr)$(sensinst)P", "$(AREA)$(DIS)$(dev):$(attr)Sts-Cte")
```

Comment:
As far as I could see, all PVs listed in the PVList are suffixed by `Prs-Cte`, which is a mix of the previous and current names.